docs/host/pypi.md

```diff
@@ -10,21 +10,90 @@ description: "The Open Web Calendar can be installed as a Python package from Py
 
 ![](https://img.shields.io/pypi/v/open-web-calendar.svg) ![](https://img.shields.io/pypi/pyversions/open-web-calendar.svg)
 
+## Installation
 The Open Web Calendar is available as a Python package on [PyPI]({{link.pypi}}).
 If you have [Python 3](https://www.python.org/) installed, run this to install the package:
 
 ```shell
 pip install open-web-calendar
 ```
+At this point you might get the error message
+```shell
+error: externally-managed-environment
+```
+
+On your productive system, it is strongly recommended to follow the instructions to create a virtual environment using 
+```shell
+python3 -m venv /home/username/venv
```

Comment:
Could you use only relative commands?

```suggestion
python3 -m .venv
source .venv/bin/activate
```

from there on, you can use the commands without the path...